klogr_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package klog_test
+
+import (
+	"bytes"
+	"flag"
+	"regexp"
+	"testing"
+
+	"k8s.io/klog/v2"
+)
+
+func TestVerbosity(t *testing.T) {
```

Comment:
Which function is this test function testing against?
for this `klog.VDepth(l.callDepth+1, klog.Level(level)).Enabled()` ？


Or is it just a missing test before? I don't see the code to be tested in this PR. Is this test any different from the previous behavior? Maybe I'm missing something.

Reply:
It's a test that was missing before, about handling of the `logr.Logger.Enabled` call. The behavior of that call changed in 1.3.0 with regards to the call depth.

If you look at the first commit in this PR, then you'll see that textlogger/textlogger.go has a change related to this.